docker-compose.manager.yml

```
@@ -0,0 +1,46 @@
+services:
+  manager:
+    image: ragapp/manager
+    build:
+      context: src/manager
+      args:
+        # Build time variables to set the basePath for static files
+        - APP_PATH=/manager
+    entrypoint: ["/app/entrypoint.sh"]
+    command: ["fastapi", "run", "main.py", "--root-path", "/manager"]
```

why do we need to pass root-path here? shouldn't traefik rewrite the URL?
```suggestion
    command: ["fastapi", "run", "main.py", "--root-path", "/manager"]
```

---

Yes, it should be rewritten by the traefik. Let me remove using rooth-path here